src/Kusto/Kusto/Kusto.csproj

```diff
@@ -14,4 +14,11 @@
     <PackageReference Include="Microsoft.Azure.Management.Kusto" Version="0.9.0-preview" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Please remove all changes from this file.

Reply:
I will change the tier in the next version (GA)